tests/unit/Collectors/DBQueryCollectorTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+use AG\ElasticApmLaravel\Agent;
+use AG\ElasticApmLaravel\Collectors\DBQueryCollector;
+use AG\ElasticApmLaravel\Collectors\RequestStartTime;
+use Codeception\Test\Unit;
+use Illuminate\Config\Repository as Config;
+use Illuminate\Database\Connection;
+use Illuminate\Database\Events\QueryExecuted;
+use Illuminate\Events\Dispatcher;
+use Illuminate\Foundation\Application;
+
+class DBQueryCollectorTest extends Unit
+{
+    private const SQL = 'SELECT * FROM user WHERE id = ?';
+    private const PROPRIETARY_SQL = 'NON_STANDARD * FROM user WHERE id = ?';
+
+    /** @var Application */
+    private $app;
+
+    /** @var Dispatcher */
+    private $dispatcher;
+
+    /** @var DBQueryCollector */
+    private $collector;
+
+    /** @var Agent|LegacyMockInterface|MockInterface */
+    private $agentMock;
+
+    /** @var Config|LegacyMockInterface|MockInterface */
+    private $configMock;
+
+    /** @var Connection|LegacyMockInterface|MockInterface */
+    private $connectionMock;
+
+    protected function _before(): void
+    {
+        $this->app = app(Application::class);
+        $this->dispatcher = app(Dispatcher::class);
+    }
+
+    protected function setUp(): void
```

Comment:
why not just use `_before` and `_after` ?

Reply:
Yes, I'm going to update all methods to be `_before` and `_after`.